### PR TITLE
Begin changing md5 to sha256

### PR DESCRIPTION
**Pull request by marco-parillo**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ Getting the ISO(Disc Image):
 
 Download it though `Get Kubuntu <http://www.kubuntu.org/getkubuntu/>`_.
```
Comment (ricktimmis) [inline docs/installation.rst]: Can we make this https

File: docs/installation.rst [hunk 2]
```diff
@@ -47,19 +47,19 @@ Run:
 
 .. code-block:: bash
 
-    md5 -r kubuntu-17.04-desktop-amd64.iso
+    sha256 -r kubuntu-17.04-desktop-amd64.iso
```
Comment (ricktimmis) [inline docs/installation.rst]: Let's change this to 20.04

File: docs/installation.rst [hunk 3]
```diff
@@ -47,19 +47,19 @@ Run:
...
 From Ubuntu Linux
 ``````````````````
 
-The first thing to do is to make sure you're in the directory with the .iso file (most likely in the Downloads directory). For more complex use of MD5SUM check this `out <https://help.ubuntu.com/community/HowToMD5SUM>`_. Then running the command 'md5sum':
+The first thing to do is to make sure you're in the directory with the .iso file (most likely in the Downloads directory). For more complex use of SHA256SUM check this `out <https://ubuntu.com/tutorials/how-to-verify-ubuntu#5-verify-the-sha256-checksum>`_. Then running the command 'md5sum':
 
 .. code-block:: bash
 
-    md5sum kubuntu-17.04-desktop-amd64.iso
+    sha256sum kubuntu-17.04-desktop-amd64.iso
```
Comment (ricktimmis) [inline docs/installation.rst]: Same here 20.04

**Review by ricktimmis — CHANGES_REQUESTED** [review]
Made a few suggestions, this is a good start

**Comment by ahoneybun** [comment]
@ricktimmis I made the changes that you requested so I think we're good for another review before merging.

**Review by ricktimmis — APPROVED** [review]
I checked the destination for the link, and it looks like a good place to put the user.
LGTM
Ship It!

**Pull request merged**